exporter/containerimage/export.go

```diff
@@ -301,7 +301,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source,
 			if e.push {
 				err := e.pushImage(ctx, src, sessionID, targetName, desc.Digest)
 				if err != nil {
-					return nil, err
+					return nil, fmt.Errorf("failed to push %v: %w", targetName, err)
```

Comment:
We don't allow `fmt.Errorf` in build repos because they lose stacktraces. Use `pkg/errors.Wrap`

Reply:
Good to know, thanks!

Reply:
Does it lose stack traces even with `%w`?

Reply:
Yes. Stdlib `errors` does not capture a stacktrace.

Reply:
Ah, makes sense. I misunderstood your comment and thought you meant it would remove existing stack traces from the error chain.